extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -12,8 +12,12 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 
 	win.ads = win.ads || {};
 	win.ads.runtime = win.ads.runtime || {};
-	win.ads.runtime.disableBtf = false;
-	win.ads.runtime.unblockHighlyViewableSlots = false;
+	// FIXME AdEng to decide how to disable these slots properly
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)